relay/implementations/redis.ts

```diff
@@ -505,7 +506,27 @@ export class Redis implements IRedis {
     return keys.length;
   }
 
-  async saveDeposit(index: number, deposit: BlsDepositData): Promise<void> {
+  async getDepositsIndices(): Promise<string[]> {
+    await this.waitForConnection();
+
+    const pattern = `${CONSTANTS.depositKey}:*`;
+    const keys = (await this.client.keys(pattern)).filter(
+      key => !key.includes(CONSTANTS.validatorRegistryLimit.toString()),
+    );
+
+    if (keys.length === 0) {
+      return [];
+    }
+
+    return (await this.client.mget(keys))?.map(
+      data => JSON.parse(data || '').deposit.depositIndex,
+    );
+  }
```

Comment:
We don't need to keep deposit indices in redis

Reply:
I think this is needed because sometimes you get an event for something you already have in redis

Reply:
The Rust workers will consume the indices for the circuits, so they are indeed needed. In the script, they are used to prevent duplicates.

Reply:
Yes, excuse me, you obviously need the indices, what I meant to say is that you don't need to diff the indices to extract the new ones, with the assumption that you can't get the same event multiple times, but apparently this is not the case